src/main/java/teammates/common/datatransfer/FeedbackParticipantType.java

```diff
@@ -141,4 +141,25 @@ public String getDisplayNameGiver() {
     public String getDisplayNameRecipient() {
         return displayNameRecipient;
     }
+    
+    /**
+     * @return A user-friendly {@FeedbackParticipantType} representing participant list.
+     */
+    public static List<FeedbackParticipantType> getParticipantListFromParams(
+            String getParticipantListFromCommaSeparatedValues) {
```

Comment:
@wkurniawan07 If i do that i need to change the method name in `*Action` classes also. Doesn't it affect ?